.github/workflows/release.yml

```diff
@@ -48,6 +55,12 @@ jobs:
           known_hosts: ${{ secrets.KNOWN_HOSTS }}
           if_key_exists: replace
 
+      # Run pre-requisites to build both applications (iOS and Android)
+      - name: Analyze and run tests
+        run: |
+          flutter analyze .
+          flutter test test
```

Comment:
Moves `analyze` and `tests` from both targets Fastfiles to the workflow. The analyze and tests result don't change based on the platform, so it was a unnecessary duplicate work